networking.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: page
+title: Making HTTP Networking Requests in Flutter
+permalink: /networking/
+---
+
+This page describes how to make HTTP networking requests in Flutter. For
+HTTP servers and sockets, see [dart:io][dartio].
+
+* TOC Placeholder
+{:toc}
+
+## Using the `http` package
+
+Flutter supports the [http package][http], version `0.11.3+12` or later.
+
+The first step is to declare a dependency by adding an `http...` line under `dependencies` in
+`pubspec.yaml`:
+
+```
+dependencies:
+  ...
+  http: '>=0.11.3+12'
+```
+
+## Making HTTP requests
+
+Next, create an HTTP [Client][client] with the `Client` constructor:
+
+<!-- skip -->
+```dart
+import 'package:http/http.dart' as http;
+
+var httpClient = new http.Client();
```

Comment:
We should recommend that folks use https://docs.flutter.io/flutter/services/createHttpClient.html .  It's useful to send all the client creation through that point so you can mock it during testing, etc.